components/KiboCollectedProduct.vue

```diff
@@ -12,12 +12,18 @@
         </component>
       </div>
       <div class="kibo-collectedProduct__price-wrapper">
-        <SfPrice
+        <p>{{ typeof specialPrice }}</p>
+        <KiboPrice
           v-if="regularPrice"
-          :regular="regularPrice"
-          :special="specialPrice"
+          :regular="$n(regularPrice, 'currency')"
+          :special="specialPrice && $n(specialPrice, 'currency')"
           class="kibo-collectedProduct__price"
+          :small="true"
+          :coupons="couponsApplied"
         />
+        <ins v-if="specialPrice" class="kibo-promo-success">
+          {{ `${couponsApplied ? "'" + couponsApplied.join(" , ") + "'" : ""}` }} promo code applied
```

Comment:
For "promo code applied", can we avoid hard code values and use localization here?

Reply:
Done